commands/build.go

```diff
@@ -138,7 +138,7 @@ func parseBuildArgs(args []string) (map[string]string, error) {
 func runBuild(cmd *cobra.Command, args []string) error {
 
 	var services stack.Services
-	if len(yamlFile) > 0 {
+	if len(yamlFile) > 0 && len(functionName) == 0 {
```

Comment:
I would rather not affect the build command, the way to filter when building is to pass `--filter FN_NAME`